peridot/db/psql/project.go

```diff
@@ -264,6 +268,7 @@ func (a *Access) UpdateProject(id string, project *peridotpb.Project) (*models.P
 		TargetVendor:       project.TargetVendor,
 		AdditionalVendor:   project.AdditionalVendor.Value,
 		Archs:              project.Archs,
+		BuildPoolType:      project.BuildPoolType,
```

Comment:
If you just change this to `project.BuildPoolType.Value`, then we should be all set

Reply:
updated